spyder/plugins/completion/kite/widgets/calltoaction.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+from qtpy.QtCore import Qt, QTimer, QUrl
```

Comment:
Also @naman I think there are some imports that are not needed anymore (like QTimer, QUrl, QDesktopServices, etc). Please remove them